packages/babel/rollup.config.mjs

```diff
@@ -0,0 +1,18 @@
+import { readFileSync } from 'fs';
+
+import { createConfig } from '../../shared/rollup.config.mjs';
+
+import { babel } from './src/index.js';
+
+const pkg = JSON.parse(readFileSync(new URL('./package.json', import.meta.url), 'utf8'));
+
+export default {
+  ...createConfig({ pkg }),
+  input: './src/index.js',
+  plugins: [
+    babel({
+      presets: [['@babel/preset-env', { targets: { node: 8 } }]],
```

Comment:
Since the `package.json#engines` have been changed we could also bump this here:
```suggestion
      presets: [['@babel/preset-env', { targets: { node: 14 } }]],
```